data-otservbr-global/scripts/globalevents/customs/save_interval.lua

```diff
@@ -4,16 +4,10 @@ local function serverSave(interval)
 	end
 
 	saveServer()
-	local message = "Server save complete. Next save in %d %ss!"
-	local messageSingle = "Server save complete. Next save in %d %s!"
+	local message = string.format(SAVE_INTERVAL_CONFIG_TIME > 1 and "Server save complete. Next save in %d %ss!" or "Server save complete. Next save in %d %s!", SAVE_INTERVAL_CONFIG_TIME, SAVE_INTERVAL_TYPE)
+	Game.broadcastMessage(message, MESSAGE_GAME_HIGHLIGHT)
+	logger.info(message)
 	Webhook.sendMessage("Server save", message, WEBHOOK_COLOR_WARNING)
```

Comment:
The period save is so fast these days we should probably not even broadcast these anymore, it's just spam now.